extensions/blocks/wordads/wordads.php

```diff
@@ -80,6 +80,10 @@ public static function gutenblock_render( $attr ) {
 			return '';
 		}
 
+		if ( isset( $attr['hideMobile'] ) && $attr['hideMobile'] && $wordads->params->is_mobile() ) {
```

Comment:
Do you think this could be simplified as the following, or am I missing something?

```suggestion
		if ( ! empty( $attr['hideMobile'] ) && $wordads->params->is_mobile() ) {
```

Reply:
Ah yeah, that's def a simpler way 👍